### DISCUSSION TITLE
Install app from store but do not start it immedately

### DISCUSSION THREAD

**Discussion by k0stett** [IceWhaleTech/CasaOS#2062] [Q&A]
Hi. How to install application from application store without running this application after successful installation?
I want to import my old Immich database backup and I must do that without starting immich-server container, only immich-postgres must be running for DB restore ([details link](https://immich.app/docs/administration/backup-and-restore/)).
Install, then start all and stop will not work for this case, I need install, start only one container from all in compose, then do my buisness and then start other containers.

**Comment by SamuelFe** [comment]
I was just looking for some way to do exatly that. It's been a year since so... have you succeded on that? I'm using the big-bear-casaos's Immich app that already packs a postgres service an I would like to install it without everithing start automatically.

**Reply by SamuelFe** [reply]
Came back to say that the [Immich docs](https://docs.immich.app/administration/backup-and-restore/) tell us to set the environment variable `DB_SKIP_MIGRATIONS=true` so that we can avoid problems with restoring the database on situations like we're having. That should be enough.
